### ISSUE TITLE
OpenPose-Multi-Person

### ISSUE THREAD

**Issue by cdebadri** [spmallick/learnopencv#167]
Thank You, for the great work!
I have a question. In OpenPose Multi-Person, the image whose keypoints are to be detected is scaled to width 368. Can I use other images of any size without resizing?

**Comment by vikasguptaiisc** [comment]
Yes, you can use other image sizes. The processing time will take a hit in that case.
